GillianCore/logging/databaseReporter.ml

```diff
@@ -6,7 +6,8 @@ let initialize () =
   LogDatabase.reset_db ();
   LogDatabase.create_db ()
 
-let will_log (type_ : string) = type_ <> LoggingConstants.ContentType.debug
+let will_log (type_ : string) =
+  not (List.mem type_ LoggingConstants.ContentType.[ debug; phase ])
```

Comment:
same, try avoiding to build lists for no reason
Also, linked list is the de-facto data structure in ocaml, but it's slower than arrays (cache, pointer indirection etc...)
So in that case I would even prefer
```
not (Array.mem type_ [| debug; phase |])
```
but in any case I would prefer neither of those :p

Reply:
Resolved in e2b58255